coderd/database/queries/workspaceagentstats.sql

```diff
@@ -332,9 +333,11 @@ WITH agent_stats AS (
 		agent_id,
 		workspace_id,
 		coalesce(SUM(rx_bytes), 0)::bigint AS rx_bytes,
-		coalesce(SUM(tx_bytes), 0)::bigint AS tx_bytes
+		coalesce(SUM(tx_bytes), 0)::bigint AS tx_bytes,
+		coalesce(MAX(connection_median_latency_ms), 0)::float AS connection_median_latency_ms
 	FROM workspace_agent_stats
-	WHERE workspace_agent_stats.created_at > $1
+	-- The greater than 0 is to support legacy agents that don't report connection_median_latency_ms.
+	WHERE workspace_agent_stats.created_at > $1 AND connection_median_latency_ms > 0
```

Comment:
This is another fix I found along the way where we weren't filtering out the stats with 0 or selecting only recent stats.